tests/test_integration.py

```diff
@@ -1,26 +1,20 @@
 from unittest import mock
```

Comment:
Putting the comment here but referring to the end-to-end test:

Now that we're not relying on Trello, we should be able to clear out all of the Trello/requests specific stuff. We can then replace what's necessary - Mongo is pretty good at creating what's needed when needed so this will probably just involve:
* In `app_with_temp_board`, setting the MONGO_DB variable to some fake test db name before we create the app
```
    os.environ['MONGO_DB'] = 'fake'
```
* Dropping the DB after we've finished, with something like:
```
    mongo_url = os.getenv('MONGO_URL')
    mongo_db = os.getenv('MONGO_DB')

    client = pymongo.MongoClient(mongo_url)
    client.drop_database(mongo_db)
```